playbooks/besman-counterfit-0.0.1-playbook.sh

```diff
@@ -20,7 +20,17 @@ function __besman_init() {
             flag=true
         fi
     done
-
+    
+    local status_code=$(curl -o /dev/null -s -w "%{http_code}\n" $BESMAN_ARTIFACT_URL)
+    if [ "$status_code" -ne 200 ]; then
+       __besman_echo_red "The $BESMAN_ARTIFACT_URL is not found."
+       __besman_echo_red "Create the model repository on github/gitlab and try again."
+       __besman_echo_red "Make the the following files available in repository."
+       __besman_echo_red "   1. $BESMAN_ARTIFACT_NAME.h5"
+       __besman_echo_red "   2. counterfit/$BESMAN_ARTIFACT_NAME.npz"
```

Comment:
The repo need not contain a counterfit sub-directory. Just mention the file names.

Reply:
Done